pycolmap/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.10)
+project(${SKBUILD_PROJECT_NAME} VERSION ${SKBUILD_PROJECT_VERSION})
+
+set(CMAKE_CUDA_ARCHITECTURES "native")
+if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+    # Some fixes for the Glog library.
+    add_definitions("-DGLOG_NO_ABBREVIATED_SEVERITIES")
+    add_definitions("-DGL_GLEXT_PROTOTYPES")
+    add_definitions("-DNOMINMAX")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /EHsc")
+    # Enable object level parallel builds in Visual Studio.
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /MP")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP")
+endif()
+
+
+find_package(colmap REQUIRED)
+
+find_package(Python REQUIRED COMPONENTS Interpreter Development.Module)
+
+find_package(pybind11 REQUIRED)
+
+pybind11_add_module(pycolmap pycolmap/main.cc)
+target_include_directories(pycolmap PRIVATE ${PROJECT_SOURCE_DIR})
+target_link_libraries(pycolmap PRIVATE colmap::colmap freeimage::FreeImage glog::glog Ceres::ceres)
```

Comment:
Do we really need to explicitly link against freeimage glog ceres? I would have expected that these are propagated from colmap?

Reply:
They do propagate but we state them explicitly because they are directly used in pycolmap as well (not only through colmap internals).

Reply:
got it.